src/UI/SignTransactionsModals/SignWithDeviceModal/components/components/ConfirmAmount/hooks/useHandleAmountReference.ts

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+
+export const useHandleAmountReference = () => {
```

Comment:
add comment what this does
must it use state?

Reply:
Updated. 👍